.github/CODEOWNERS

```diff
@@ -1027,6 +1027,9 @@
 # ServiceLabel: %Cognitive - Anomaly Detector %Service Attention
 #/<NotInRepo>/          @yingqunpku @bowgong
 
+# ServiceLabel: %Cognitive - Content Safety %Service Attention
+#/<NotInRepo>/          @mengaims @lemonicetea
```

Comment:
I think this is for issue notification, not the code owners.

Reply:
updated